app/javascript/components/ErrorBoundary.tsx

```diff
@@ -20,19 +21,31 @@ export const ErrorMessage = ({
 }: {
   error: Error | unknown
   defaultError: Error
-}) => {
+}): null => {
   useErrorHandler(error, { defaultError: defaultError })
 
   return null
 }
 
+const handleError = (error: Error) => {
+  if (error.name === 'HandledError') {
+    return
```

Comment:
Does this swallow any errors now? As in: do errors still get reported in the console?

Reply:
This is something I want to check with Bugsnag. Currently, on development, the errors still get reported. But on production, I'm not entirely sure. I've been reading up and see that the webpack dev server raises the errors raised by the error boundary in the console.

Reply:
Aha